### ISSUE TITLE
Individual Toolbar Height(s)

### ISSUE THREAD

**Issue by hpw4711** [plashenkov/TBX#13]
I do have various toolbars & tool-windows with different heights.

The question or suggestion is to have on the current Dock and on each ROW various toolbar heights without automatic toolbar/tool-window enlarging (in height).

Example:

- Toolbar 1 has height 20
- Toolbar 2 has height 30

If both are on Dock raw 0: all toolbar heights are altered to 30.

If each toolbar is on a different raw: each toolbar has its own height as by design as 20/30

In other words how to have different toolbar heights (without automatic resize) at a given raw...
